app/shared/app-data/src/commonMain/kotlin/domain/media/cache/engine/TorrentMediaCacheEngine.kt

```
@@ -541,7 +550,16 @@ class TorrentMediaCacheEngine(
             val newMetadata = metadata.withExtra(
                 mapOf(
                     EXTRA_TORRENT_DATA to data.data.toHexString(),
-                    EXTRA_TORRENT_CACHE_DIR to downloader.getSaveDirForTorrent(data).absolutePath,
+                    EXTRA_TORRENT_CACHE_DIR to downloader.getSaveDirForTorrent(data).absolutePath.let { path ->
+                        val stripped = path.substringAfter(baseSaveDirProvider.saveDir)
+                        if (path == stripped) {
+                            logger.warn {
+                                "Failed to strip torrent save path of media ${origin.mediaId}, " +
+                                        "path: $path, base: ${baseSaveDirProvider.saveDir}"
+                            }
```

这里 silently fail 是不是会导致未来也不能播, 建议不要忽略错误